Modules/gcmodule.c

```diff
@@ -326,7 +326,7 @@ gc_get_objects_impl(PyObject *module, Py_ssize_t generation)
     }
 
     PyInterpreterState *interp = _PyInterpreterState_GET();
-    return _PyGC_GetObjects(interp, generation);
+    return _PyGC_GetObjects(interp, (int)generation);
```

Comment:
why not just change the type to int in the clinic input above?

Reply:
It gives a slightly better error message for values that don't fit in an `int`.
Since it's vanishingly unlikely for anyone to pass in that large a number, I doubt it matters.

Currently if you pass in `1 << 32` you get "generation parameter must be less than the number of  available generations", instead of the argument clinic overflow message.